### PR TITLE
Minor Fixes

### PR DESCRIPTION
**Pull request by tcyrus**
- Updated FB Logo
- Updated jQuery to latest stable (previously using a pre-release)
- Minor CSS, JS, and HTML Changes

### CODE REVIEW

File: site/static/box.js [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
+            $el.fadeOut(500, function() {
+                $el.html(things[index] + ".");
+                $el.fadeIn(500);
             });
-        }
-        else {
-            el.html("Teens for Teens.");
-            el.show();
+        } else {
+            $el.html("Teens for Teens.");
```
Comment (ovkulkarni) [inline site/static/box.js:12]: `Teens for Teens.` -> `Unified Democracy` please

Comment (tcyrus) [inline site/static/box.js:12]: @ovkulkarni, not the point of the Pull request

Comment (ovkulkarni) [inline site/static/box.js:12]: you can still do it but ok

**Comment by ovkulkarni** [comment]
@tcyrus can you reopen this pr with merging into testing

**Comment by tcyrus** [comment]
@ovkulkarni see #100

**Pull request merged**